misc/distributed_search/inner.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# set Slurm block 
+#SBATCH --job-name=alphaDIA
+#SBATCH --time=21-00:00:00
+#SBATCH --output=./logs/%A_%a_%x-slurm.out
+
+# navigate to chunk directory
+slurm_index=${SLURM_ARRAY_TASK_ID}
+chunk_directory="${target_directory}chunk_${slurm_index}/"
+cd $chunk_directory || exit
+
+# determine config file
+config_filename=$(ls | grep "config.yaml")
```

Comment:
Since this is inside automatically generated chunk directories and considering that this config is only a working copy of the user-set config.yaml, I would be in favor of removing this entirely and just setting config_filename="config.yaml"